Lombiq.Tests.UI/BasicOrchardFeaturesTesting/BasicFeaturesTestingUITestContextExtensions.cs

```diff
@@ -330,20 +399,15 @@ public static Task TestLoginAsync(
     public static Task TestLoginWithInvalidDataAsync(
         this UITestContext context,
         string userName = DefaultUser.UserName,
-        string password = "WrongPass!") =>
-        context.ExecuteTestAsync(
+        string password = DefaultUser.Password,
+        string logInButtonText = OrchardCoreLoginPage.DefaultLoginButtonText) =>
+        context.TestLoginAsync(
             "Test login with invalid data",
-            async () =>
-            {
-                await context.SignOutDirectlyAsync();
-
-                var loginPage = await context.GoToLoginPageAsync();
-                (await loginPage.LogInWithAsync(context, userName, password))
-                    .ShouldStayOnLoginPage()
-                    .ValidationSummaryErrors.Should.Not.BeEmpty();
-
-                (await context.GetCurrentUserNameAsync()).ShouldBeEmpty();
-            });
+            userName,
+            password + "WrongPass!",
+            signOut: true,
+            shouldBeSuccess: false,
+            logInButtonText);
 
     /// <summary>
```

Comment:
I just realized that in the new code, on line 468: https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI/BasicOrchardFeaturesTesting/BasicFeaturesTestingUITestContextExtensions.cs#L468

The new login button text is not applied, so it fails if you test registration.